### PR TITLE
add move prover english version

### PR DESCRIPTION
**Pull request by geometryolife**

### CODE REVIEW

File: docs/03-move/100-move-prover/02-mvp-tutorial.md [hunk 1]
```diff
@@ -1,5 +1,574 @@
...
+
+Some readers may be curious about the order of the three statements in the spec block:
+Why the definition of balance can be written after `aborts_if !exists<Balance<CoinType>>(addr)`.
+Because, if the latter holds true, `balance` does not actually exist.
+Wouldn't this order cause the Prover to fail?
+Simply put: no, the statements in the spec block are declarative and the order doesn't matter.
+
+For a more detailed understanding, you can refer to the MSL documentation for more information.
+
+### 指定 `withdraw` 的功能性质
```
Comment (whst) [inline docs/03-move/100-move-prover/02-mvp-tutorial.md:274]: do you plan to leave this part as is and translate it later?

File: docs/03-move/100-move-prover/02-mvp-tutorial.md [hunk 2]
```diff
@@ -1,5 +1,574 @@
...
+- What is the return value of this function?
+- After the function is executed, how will the state of the virtual machine be changed?
+- What invariants does this function maintain?
+
+For example, Move Prover allows all possible aborts by default when we don't give any abort conditions.
+And in the simple snippet above, we tell Prover with the directive `aborts_if_is_strict`:
+
+> I would like to strictly check the possibility of aborting this function. Report an error if there is any abort not listed by the programmer.
+
+Now, we run the attestation command in the `BasicCoin` directory:
```
Comment (whst) [inline docs/03-move/100-move-prover/02-mvp-tutorial.md:175]: `attestation` should simply be `prove`

File: docs/03-move/100-move-prover/02-mvp-tutorial.md (hunk 1 above)
Comment (geometryolife) [reply docs/03-move/100-move-prover/02-mvp-tutorial.md]: I translated everything.

File: docs/03-move/100-move-prover/02-mvp-tutorial.md [hunk 3]
```diff
@@ -1,5 +1,606 @@
...
+   =     at ./sources/BasicCoin.move:53: transfer (spec)
+   =     at ./sources/BasicCoin.move:54: transfer (spec)
+   =     at ./sources/BasicCoin.move:45: transfer
+   =         from = signer{0x0}
+   =         to = 0x0
+   =         amount = 1
+   =         _witness = <generic>
+```
+
+It is unexpectedly unexpected. Prover prompts that the rear conditions are not satisfied, indicating that the behavior described in the previous spec block is not exactly the same as the `transfer` function.
```
Comment (whst) [inline docs/03-move/100-move-prover/02-mvp-tutorial.md:388]: "unexpectedly unexpected" is a repetition. I suggest change it to "It is somewhat out of our expectation"

File: docs/03-move/100-move-prover/02-mvp-tutorial.md [hunk 4]
```diff
@@ -1,5 +1,606 @@
...
+Note that `if (P) E1 else E2` is not the same as conditional execution in program logic -- it's actually a syntactic sugar equivalent to `ensures` both `P ==> E1` and `!P ==> E2`.
+And `p ==> q` is actually `!p || q`.
+
+That is to say, the end of the second way of writing actually represents this logic:
+
+```rust
+ensures (addr_from == to  ===>  eq_post) && (addr_from != to  ===> ne_post);
+```
+
+scilicet:
```
Comment (whst) [inline docs/03-move/100-move-prover/02-mvp-tutorial.md:421]: Maybe I'm wrong but I think this is rarely used in technical documents. I suggest to use "that is", "namely", or "equivalently,".

**Comment by whst** [comment]
I went thru the whole translated file. Could you make changes to the places I commented?
Will merge it to main when everything's fine.

And much thanks for your translation!

**Comment by geometryolife** [comment]
> 

I have modified it

**Review by whst — APPROVED** [review]
Looks good. Thanks.

**Pull request merged**